doc/source/user_guide/geometrical_transform.rst

```diff
@@ -0,0 +1,132 @@
+============================================
+Geometrical transformations of images
+============================================
+
+Cropping, resizing and rescaling images
+---------------------------------------
+
+.. currentmodule:: skimage.transform
+
+Images being NumPy arrays (as described in the :ref:`numpy` section), cropping
+an image can be done with simple slicing operations. Below we crop a 100x100
+square corresponding to the top left corner of the astronaut image. Note that
```

Comment:
```suggestion
square corresponding to the top-left corner of the astronaut image. Note that
```
to be consistent with line 51 of `doc/examples/transform/plot_transform_types.py`